toolkit/tools/internal/csvparser/csvparser.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Parses the timestamps stored in csv files and print them
+// to the terminal at the end of the build.
+
+package csvparser
```

Comment:
Since this is not a generic parser but instead is hard-coded to deal only with the output form the `timestamp` tool, it should be part of the `timestamp` package as well. Ideally, I'd also re-name it, to make it clear that it is intended to work only with these very particular CSV files.